deploy.ci.acceptance.mariadb.dynamic-store-off.robot.yml

```diff
@@ -6,7 +6,10 @@ tag: '1.0'
 environment: docker.ci
 pipeline: docker.ci.acceptance.dynamic-store-off.robot
 image:
-    tag: spryker/php:8.3
+    tag: spryker/php:8.3-rc
+    php:
+        enabled-extensions:
+            - otel
```

Comment:
in all places really :)

Reply:
Yes, as we discussed, for now this extension is required